### DISCUSSION TITLE
Tags in tag cloud instead of scrollable list

### DISCUSSION THREAD

**Discussion by Cantello** [jonaswinkler/paperless-ng#1117] [Feature Requests]
I now have about ~ 25 tags and find it annoying to scroll through the list to find the one that I am looking for. Would it be possible to have the tags displayed as a cloud or as a table where horizontal space is used as well (instead of only vertical space as it is now)?
I don't know how many tags others have or what is considered useful so in case this would be only useful for me, please chime in.

**Comment by steffe** [comment]
I'm very new to paperless-ng, working now today on inserting 200 Documents, the first batch, next 200-300 to come next days. I've 18 Tags and experienced the same. Would be great to have the tags all or the most used once visible without scrolling.
